internal/commands/vaultsecrets/secrets/displayer.go

```diff
@@ -100,16 +105,92 @@ func (d *displayer) secretsFieldTemplate() []format.Field {
 		})
 	}
 
+	fields = append(fields, format.Field{
+		Name:        "Created At",
+		ValueFormat: "{{ .CreatedAt }}",
+	})
+
+	switch d.secretType {
+	case secretTypeKV, secretTypeRotating:
+		fields = append(fields, format.Field{
+			Name:        "Latest Version",
+			ValueFormat: "{{ .LatestVersion }}",
+		})
+	}
+
 	return fields
 }
 
 func (d *displayer) openAppSecretsFieldTemplate() []format.Field {
-	fields := d.secretsFieldTemplate()
-	return append(fields, d.fields...)
+	fields := []format.Field{
+		{
+			Name:        "Secret Name",
+			ValueFormat: "{{ .Name }}",
+		},
+		{
+			Name:        "Type",
+			ValueFormat: "{{ .Type }}",
+		},
+	}
+
+	// Secret type specific fields
+	switch d.secretType {
+	case secretTypeKV:
+		fields = append(fields, []format.Field{
+			{
+				Name:        "Created At",
+				ValueFormat: "{{ .CreatedAt }}",
+			},
+			{
+				Name:        "Latest Version",
+				ValueFormat: "{{ .LatestVersion }}",
+			},
+			{
+				Name:        "Value",
+				ValueFormat: "{{ .StaticVersion.Value }}",
+			},
+		}...)
+	case secretTypeDynamic:
+		fields = append(fields, []format.Field{
+			{
+				Name:        "Created At",
+				ValueFormat: "{{ .DynamicInstance.CreatedAt }}",
+			},
+			{
+				Name:        "Expires At",
+				ValueFormat: "{{ .DynamicInstance.ExpiresAt }}",
+			},
+			{
+				Name:        "Time-to-Live",
+				ValueFormat: "{{ .DynamicInstance.TTL }}",
+			},
+			{
+				Name:        "Values",
+				ValueFormat: fmt.Sprintf(multiValueSecretFmt, ".DynamicInstance.Values"),
+			},
+		}...)
+	case secretTypeRotating:
```

Comment:
@averche - Let me know if there are any rotation-specific fields we can add to either read or open commands!

Reply:
Thanks for tagging! I think adding the `Expires At` field to match dynamic secrets would be great! We could also consider adding the `Created At` for both.

Reply:
Np! Added in 312c378cdadbfa633234fe5e92c4b06b8a1ee82e.